docs/cookbook/deployment/sylius-cloud-management-basics.rst

```diff
@@ -0,0 +1,232 @@
+Sylius Cloud by Platform.sh management basics
+=============================================
+
+Creating a new project and environment
+--------------------------------------
+
+Every new environment needs to be created in the Sylius Cloud Console or by using the CLI.
```

Comment:
cloud console and CLI it's the same place, if i understand correctly, you mean maybe the console and ui interface of the platform.sh site